### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Azure Activity log_Screen Reader: Narrator is not reading the name for the ‘Azure Activity Log’ and ‘Azure Explorer’ tab item when user presses arrow keys to navigate

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3348]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #Narrator

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator
MAS Violated: MAS1.3.1

**Note: Similar issue is found for ‘Azure Explorer’ tab item**

**Repro Steps:** 
1) Start Narrator and Open Java EE - Eclipse application 
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Select any app service and tab till Deploy and press enter
11) Tab till ‘Azure Activity Log’ 
12) Verify if narrator is reading the name of the tab items

**Actual Result:** 
When focus is on ‘Azure explorer’ tab item and user presses arrow key to navigate to the ‘Azure Activity log’ narrator is not reading where exactly the focus is (when navigating between these two tab items, keyboard focus is moving but narrator is not reading the focused tab item)

**Expected Result:** 
Narrator must read the name of the tab item along with the status when focus lands on any tab item
Eg: ‘Azure Activity log’ tab 1 of 2 selected

**User Impact:** 
Screen reader users will find difficulty as where exactly the focus is if narrator is not reading the name of the tab items

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_21_narrator is not reading the tab names.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451326/Bug_21_narrator.is.not.reading.the.tab.names.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati The behavior of tool window is controlled by Eclipse, which is out control of our plugin,  so could you please open an issue in eclipse(https://bugs.eclipse.org/bugs/). Here are steps to verify this issue:
1. Open Show view, select some views to make them shown in Eclipse tool window.(For instance, Problems, Java Doc)
2.Tab till the tag you added in last step
3. Verify if narrator is reading the name of the tab items when user presses arrow key to navigate to other tabs.

**Comment by Flanker32** [comment]
- Importance of experience: Little, it's not a common behavior for users to switch to toolwindow, we hadn't received any customer feedback about this issue.
- Quality of experience: Medium, users may not know the tab name if narrator is not reading the name of the tab items
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Tanushagottipati  This is an external issue of Eclipse which has been linked to [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), please help us review it, thanks!

**Comment by Tanushagottipati** [comment]
External Bug:https://bugs.eclipse.org/bugs/show_bug.cgi?id=551116

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749750
